android/app/src/main/java/com/yoda/MainApplication.java

```diff
@@ -1,5 +1,6 @@
 package com.yoda;
 
+import com.evollu.react.fcm.FIRMessagingPackage;
```

Comment:
You seemed to misunderstand...
In MainActivity, this package's import is not necessary.